content/en/docs/setup/independent/high-availability.md

```diff
@@ -223,6 +223,12 @@ SSH is required if you want to control all nodes from a single machine.
     done
     ```
 
+{{< caution >}}
+Copy only the certificates in the above list. kubeadm will take care of generating the rest of the certificates
+with the required SANs for the joining control-plane instances. If by mistake you copy all the certificates,
```

Comment:
```suggestion
with the required SANs for the joining control-plane instances. If you copy all the certificates by mistake,
```

Not a native, wouldn't this rephrase work better?

Reply:
sgtm